### ISSUE TITLE
CSV File input : auto-typing should be optional

### ISSUE THREAD

**Issue by simonaubertbd** [amphi-ai/amphi-etl#154]
Hello,

As of today, when using CSV File input, types are automatically set by Amphi. I would like to deactivate this behaviour as an option, in order to control myself the types and avoid bad typing.

Best regards,

Simon

**Comment by tgourdel** [comment]
Thanks Simon, that's a very valid point. I think you are very much right and plan to add the option in the file inputs.
However, in order to understand better, could you tell me a few examples where the automated mapping isn't providing the right expectations? 
Thanks a lot

**Comment by simonaubertbd** [comment]
@tgourdel  Yep, sure.  Most of time, it's because I want to keep it as string, such as this example
![image](https://github.com/user-attachments/assets/97f894bd-3836-4310-a4c1-5d70879e8e47)
With this sample, Amphi will transform the model and the version in numbers.
![image](https://github.com/user-attachments/assets/20f05cc7-e6ff-4ef1-89f9-8d84a0a6b74f)
 But what if I make after a union with another brand ? it won't work
Best regards,

Simon

**Comment by tgourdel** [comment]
Merci! :)
